src/stylesheets/components/_status-tag.scss

```diff
@@ -0,0 +1,3 @@
+a:hover .app-c-tag__status {
```

Comment:
The _component (or block)_ is a tag, with a status _modifier_. So this file should be called `_tag.scss`.

Reply:
Fixed